runtimes/spiritnet/src/lib.rs

```diff
@@ -690,6 +694,111 @@ impl pallet_utility::Config for Runtime {
 
 impl pallet_randomness_collective_flip::Config for Runtime {}
 
+// Start of proxy pallet configuration
+
+parameter_types! {
+	// One storage item; key size 32, value size 8; .
+	pub const ProxyDepositBase: Balance = KILT;
+	// Additional storage item size of 33 bytes.
+	pub const ProxyDepositFactor: Balance = KILT;
+	pub const MaxProxies: u16 = 32;
+	pub const AnnouncementDepositBase: Balance = KILT;
+	pub const AnnouncementDepositFactor: Balance = KILT;
+	pub const MaxPending: u16 = 32;
+}
+
+/// The type used to represent the kinds of proxying allowed.
+#[derive(
+	Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, RuntimeDebug, MaxEncodedLen, scale_info::TypeInfo,
+)]
+pub enum ProxyType {
+	Any,
+	NonTransfer,
+	Governance,
+	Staking,
+	CancelProxy,
+	Ctype,
+	Attestation,
+	Delegation,
+	Did,
+}
+
+impl Default for ProxyType {
+	fn default() -> Self {
+		Self::Any
+	}
+}
+
+impl InstanceFilter<Call> for ProxyType {
+	fn filter(&self, c: &Call) -> bool {
+		match self {
+			ProxyType::Any => true,
+			ProxyType::NonTransfer => matches!(
```

Comment:
One of our collators wanted to have the Proxy Pallet specifically for staking.

Reply:
I added parachain staking to the NonTransfer group, but the did operations are now completely out because as @Diiaablo95 pointed out, it doesnt make sense to have a proxy for those because they are already proxied through the did pallet

Reply:
Could you do the same for peregrine & standalone? I would also add the did pallet. There some use cases which could benefit from it.